addon/webextension/content-script.js

```diff
@@ -40,14 +40,14 @@ function findAndReplace(wordList) {
     node.appendChild(hoverEle);
   });
 
-    // append the ones we saw as a result
-    if (document.querySelector('#wanted')) {
-      document.querySelector('#wanted').innerText="wanted: " + Array.from(wordList);
-    }
-    if (document.querySelector('#wanted')) {
-      document.querySelector('#seen').innerText="seen: " + Array.from(seen);
-    }
+  // append the ones we saw as a result
+  if (document.querySelector("#wanted")) {
+    document.querySelector("#wanted").innerText = "wanted: " + Array.from(wordList);
   }
+  if (document.querySelector("#wanted")) {
```

Comment:
@gregglind Should this be `.querySelector("#wanted")` or `.querySelector("#seen")`?

Reply:
As it turns out, it's immaterial.